mlflow/server/js/src/sdk/MlflowMessages.js

```diff
@@ -112,15 +112,6 @@ export const RunInfo = Immutable.Record({
   // optional INT64
```

Comment:
Were you able to autogenerate these changes from the proto? Or did you have to make them manually?

Reply:
... I made them manually, to help confirm w/the JS compilation that I had removed all references.